cityofaustin/atd-vz-data
# Remove the `_lkp` suffix from lookup tables

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/18569

## Testing

You know the deal! Run the CRIS import into your local stack and give the app a whirl:
1. Advanced search the crashes list
2. Export the crashes list
3. Edit a unit and people records using the drop-downs. Edit all the data table cards you can find.
4. Create a temp crash record
5. Search the locations list and export it
6. Open a location details and add a note
7. Check out the fatalities view

---
#### Ship list
